packages/gatsby-transformer-sharp/src/extend-node-type.js

```diff
@@ -154,10 +154,22 @@ const fixedNodeType = ({
         type: ImageCropFocusType,
         defaultValue: sharp.strategy.attention,
       },
+      fit: {
+        type: ImageFitType,
+        defaultValue: sharp.fit.cover,
+      },
+      background: {
+        type: GraphQLString,
+        defaultValue: `rgba(0,0,0,1)`,
+      },
       rotate: {
         type: GraphQLInt,
         defaultValue: 0,
       },
+      trim: {
+        type: GraphQLFloat,
+        defaultValue: false,
```

Comment:
Let's instead check that it hasn't been passed. It doesn't feel right to have different default value type than the actual type (actually pretty surprised it even works on GraphQL side)

Reply:
@freiksenet As I mentioned, GraphQL coalesces false and true to 0 and 1 respectively.

I can just change it to 0, which is effectively the same.